testing/tests/ui/let_destructuring_has_rest.stderr

```diff
@@ -0,0 +1,26 @@
+error: unexpected `,` character after `..`
+       failed to parse template source at row 2, column 20 near:
+       ", } = x -%}hello {{ a }}{%- endif -%}\n"
+  --> tests/ui/let_destructuring_has_rest.rs:16:10
+   |
+16 | #[derive(Template)]
+   |          ^^^^^^^^
+   |
+   = note: this error originates in the derive macro `Template` (in Nightly builds, run with -Z macro-backtrace for more info)
+
+error: failed to parse template source at row 2, column 17 near:
```

Comment:
This error message got worse, should precise that a comma is missing.

Reply:
Absolutely right. I added a new helper function `collect_targets` that has better error messages likely errors such as missing a comma.